plugin/newebuild.vim

```diff
@@ -139,10 +139,10 @@ fun! <SID>MakeNewEbuild()
             put =''
             put ='CP_DEPEND=\"\"'
             put =''
-            put ='RDEPEND=\">=virtual/jre-1.8:*'
-            put ='	${CP_DEPEND}\"'
-            put ='DEPEND=\">=virtual/jdk-1.8:*'
-            put ='	${CP_DEPEND}\"'
+            put ='DEPEND=\"${CP_DEPEND}'
+            put ='	\|\| ( virtual/jdk:1.8 virtual/jdk:11 virtual/jdk:17 virtual/jdk:21 )\"'
```

Comment:
Shouldn't you start from the newest? Also, this is getting quite long, so I suppose wrapping it into multiple lines would improve readability.

Reply:
Thanks for the review.  Splitting lines indeed makes makes it much easier to drop single entries.